config/crd/bases/primaza.io_servicebindings.yaml

```diff
@@ -20,6 +20,9 @@ spec:
       jsonPath: .status.state
       name: State
       type: string
+    - jsonPath: .metadata.annotations.primaza\.io/registered-service-name
```

Comment:
Why only one dot (`\.`) is escaped but not the other three dots?

Reply:
Because the annotation key is "primaza.io/registered-service-name"